packages/template-retail-react-app/app/pages/account/wishlist/index.jsx

```diff
@@ -137,7 +137,7 @@ const AccountWishlist = () => {
         return isValidChange
     }
 
-    const hasWishlistItems = wishListItems?.length > 0
+    const hasWishlistItems = Object.keys(wishListItems ?? {}).length > 0
```

Comment:
I see.. `wishListItems` is no longer an array. It's now an object.

Ok let me test-drive the PR.